docs/usage/index.md

```diff
@@ -129,6 +129,7 @@ undefined
 .. toctree::
    :hidden:
 
+   working-with-bundlers.md
```

Comment:
Let's put this in the bottom of this tree.

![image](https://github.com/pyodide/pyodide/assets/24893111/6528c782-ec16-4aee-a3a2-028d0bd4240e)